frontend/storybook/src/components/entry/homepage/aboutus-paragraph.ts

```diff
@@ -0,0 +1,112 @@
+import {argsToAttrs, deleteNone} from "@utils/attributes";
+import "@elements/entry/home/TOSORT/icon-wparagraph";
+import "@elements/entry/home/sections/aboutus-section";
+import "@elements/core/titles/variants/title-section";
+import "@elements/core/titles/variants/subtitle";
+import "@elements/entry/home/TOSORT/list-type";
+import "@elements/core/dividers/circle-div";
+import "@elements/core/titles/title-sub-paragraph";
+import {Color, FontSize} from "@elements/core/titles/title-sub-paragraph";
+
+
+  
+export default {
+  title: 'Homepage',
+}
+
+interface Args{
+  color: Color,
+  size: FontSize,
+}
+
+const DEFAULT_ARGS:Args = {
+  color:"yellow",
+  size: "medium",
+}
+
+
+const STR_STARTTITLE = "What they say ";
+const STR_ENDTTITLE = "about us ";
+
+const STR_TITLEPARENTS = "Parents";
+const STR_SUBTITLEPARENTS = "Sarah Nazirah, Mexico";
+
+const STR_TITLETEACHERS = "Teachers";
+const STR_SUBTITLETEACHERS = "Sarah Nazirah, Mexico";
+
+const STR_PINK = "pink";
+const STR_SIZELARGE = "large";
+const STR_LINE1="I want to tell you, because of JI, my children are";
+const STR_LINE2="learning Hebrew and English simultaneously. For my ";
+const STR_LINE3="children, you are not only teaching two children, you"
+const STR_LINE4="are also saving their souls. I reaffirm that they have";
+const STR_LINE5="achieved educational rehabilitation, thanks to JI.";
+const STR_YELLOW="yellow";
+const STR_BLACK="black";
+const STR_SMALL="small";
+const STR_WHITE="white";
+const STR_MEDIUM="medium";
+const STR_SARAHN="sarah-nazirah.png";
```

Comment:
icon paths and color properties should not be in STR_ prefixes. The intent of the STR_ prefixes is that we will eventually move them out of code completely and they will be set in a database for copywriters/translators.

For story mock data, just set the default properties in DEFAULT_ARGS directly (and other radio options in `argTypes`